autopilot-daemon/pkg/handlers/healthchecks.go

```diff
@@ -12,16 +12,27 @@ import (
 	"k8s.io/klog/v2"
 )
 
+var defaultPeriodicChecks string = "pciebw,remapped,dcgm,ping,gpupower"
+
 func TimerRun() {
 	klog.Info("Running a periodic check")
-	runAllTestsLocal("all", "pciebw,remapped,dcgm,ping,gpupower", "1", "None", "None", nil)
+	checks, exists := os.LookupEnv("PERIODIC_CHECKS")
+	if !exists {
+		klog.Info("Run all periodic health checks\n")
+		checks = defaultPeriodicChecks
+	}
+	runAllTestsLocal("all", checks, "1", "None", "None", nil)
 }
 
 func runAllTestsLocal(nodes string, checks string, dcgmR string, jobName string, nodelabel string, r *http.Request) (*[]byte, error) {
 	out := []byte("")
 	var tmp *[]byte
 	var err error
 	start := time.Now()
+	if strings.Contains(checks, "all") {
+		klog.Info("Run all periodic health checks\n")
+		checks = defaultPeriodicChecks
```

Comment:
I don't know if it's ok to set `all` to what's in the periodic.. it could be ok but I don't know if that's the right purpose of `all`. We can leave it this way for now..
Anyway can you remove the print above? That is not right, it's not a periodic check

Reply:
I've reverted the commit back to just the env variable change. We can clear up the all ambiguity in another issue :)